# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../powerpc-utils/hotfix.patch
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pmac-utils/Makefile.vanilla	2017-04-23 15:19:56.316919669 +0000
+++ pmac-utils/Makefile	2017-04-23 15:20:09.004919886 +0000
@@ -27,7 +27,7 @@
 INSTALL	= /usr/bin/install -c
 SOUND_INC = -I.
 
-PROGS	= clock fdeject mousemode nvsetenv nvvideo sndvolmix vmode
+PROGS	= fdeject mousemode nvsetenv nvvideo sndvolmix
 SCRIPTS	= macos
 
 # DEPENDENCIES:
